tikv/raftstore/peer_worker.go

```diff
@@ -163,29 +168,33 @@ func (rw *raftWorker) receiveMsgs(closeCh <-chan struct{}) (quit bool) {
 	var reqCount int
 	var respCount int
 	var raftMsgCount int
+	var receivingTime time.Time
 	select {
 	case <-closeCh:
 		for _, applyCh := range rw.applyChs {
 			applyCh <- nil
 		}
 		return true
 	case msg := <-rw.raftCh:
+		receivingTime = time.Now()
 		reqCount++
 		if msg.Type == MsgTypeRaftMessage {
 			raftMsgCount++
 		}
 		rw.getPeerInbox(msg.RegionID).append(msg)
 	case msg := <-rw.applyResCh:
+		receivingTime = time.Now()
 		respCount++
 		rw.getPeerInbox(msg.RegionID).append(msg)
 	case <-rw.ticker.C:
+		receivingTime = time.Now()
```

Comment:
It would be simpler to get the time after the `select` block.

Reply:
Fixed it.